example-adapter/build.gradle.kts

```diff
@@ -40,20 +40,28 @@ dependencies {
 }
 
 group = "org.niis"
-version = "0.0.7"
+version = "0.0.8-SNAPSHOT"
```

Comment:
I think this needs to be `0.0.8` in the master branch and then bumped to `0.0.9-SNAPSHOT` in develop later.